trailblazer/apps/slurm/api.py

```diff
@@ -1,10 +1,46 @@
-from typing import Callable, Dict, List, Optional
+import subprocess
+from pathlib import Path
+from typing import Any, Callable, Dict, List, Optional
 
 from trailblazer.apps.slurm.models import SqueueResult
 from trailblazer.apps.slurm.utils import formatters
 from trailblazer.constants import FileFormat, SlurmJobStatus, TrailblazerStatus
 from trailblazer.exc import EmptySqueueError
-from trailblazer.io.controller import ReadStream
+from trailblazer.io.controller import ReadFile, ReadStream
+
+
+def _get_squeue_jobs_flag_input(slurm_job_id_file_content: Dict[str, List[str]]) -> str:
+    """Return SLURM squeue jobs flag formatted input."""
+    job_ids: List[str] = slurm_job_id_file_content.get(next(iter(slurm_job_id_file_content)))
+    return ",".join(map(str, job_ids))
+
+
+def get_slurm_squeue_output(slurm_job_id_file: Path) -> Any:
+    """Return squeue output from ongoing analyses in SLURM."""
+    slurm_job_id_file_content: Dict[str, List[str]] = ReadFile.get_content_from_file(
+        file_format=FileFormat.YAML, file_path=slurm_job_id_file
+    )
+    slurm_jobs: str = _get_squeue_jobs_flag_input(
+        slurm_job_id_file_content=slurm_job_id_file_content
+    )
+    return (
+        subprocess.check_output(
+            [
+                "ssh",
+                "hiseq.clinical@hasta.scilifelab.se",
```

Comment:
with these changes we are always doing ssh to hasta, is that what we want?

Reply:
No, I misread the code and this would have broken it. Reverted 👍